helper/EventHelper.go

```diff
@@ -594,6 +594,8 @@ type ScanEvent struct {
 	Token               string `json:"token"`
 	AwsRegion           string `json:"awsRegion"`
 	DockerRegistryId    string `json:"dockerRegistryId"`
+	DockerConnection    string `json:"dockerConnection"`
```

Comment:
what is use of docker connection ?

Reply:
if it referring to insecure then please rename it properly and consistently

Reply:
we are using `DockerConnection` everywhere, even in commonWorkflowRequest, as it can have values, `insecure`, `secure-with-cert`